language-support/java/bindings-rxjava/src/test/java/com/daml/ledger/rxjava/components/TemplateA.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I would be great to use some generated Template java code, but I don't have any idea how to do that...